### DISCUSSION TITLE
Access to interactive documents(Swagger UI) request header, date format issues

### DISCUSSION THREAD

**Discussion by IwasIverson** [fastapi/fastapi#7409] [Questions]
This is the result of  Request header to access the Swagger UI : 
date: Wed05 Aug 2020 04:30:17 GMT

In fact, It may be like this：
date: Wed 05 Aug 2020 04:30:17 GMT

Is there a space missing between 'Wed' and '05'

**Comment by tiangolo** [comment]
Sorry, I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.
